modules/nf-core/gatk4spark/applybqsr/tests/nextflow.config

```diff
@@ -0,0 +1 @@
+docker.userEmulation = true
```

Comment:
this is getting deprecated, I don't think we should do that...

Reply:
I'm happy to fix it another way if you have one 😓

Reply:
@nf-core/maintainers does anyone know what this option does exactly? It seems to be the only that gets spark going

Reply:
It's been documented [here](https://github.com/nextflow-io/nextflow/discussions/4601), but there has to be more going on than that

Reply:
from git, `docker.userEmulation` add this to the command
```
'-u $(id -u) -e "HOME=${HOME}" -v /etc/passwd:/etc/passwd:ro -v /etc/shadow:/etc/shadow:ro -v /etc/group:/etc/group:ro -v $HOME:$HOME'
```

Reply:
Exactly. Update `docker.runOptions` with what you found on Github. And add the group too.

Reply:
Actually I'm not sure if some of that should be split between `runOptions` and `containerOptions`.

Reply:
Adding the whole string to `runOptions` fixed the issue, so I think I'll leave it at that for now